### PR TITLE
Add/circleci config

### LINKED ISSUE #100: Set up Circle/TravisCI

### PR DESCRIPTION
**Pull request by knowbody**
closes #100

### CODE REVIEW

**Comment by czystyl** [comment]
Only test on node 8 ?

**Comment by knowbody** [comment]
that's the latest LTS version

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    working_directory: ~/mern-starter
+    docker:
+      - image: circleci/node:8.11.2
+    steps:
+      - checkout
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@latest'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
```
Comment (czystyl) [inline .circleci/config.yml]: If you only add a scrtips without changing dependencies the cache will go away

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+jobs:
+  build:
+    working_directory: ~/mern-starter
+    docker:
+      - image: circleci/node:8.11.2
+    steps:
+      - checkout
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@latest'
```
Comment (czystyl) [inline .circleci/config.yml]: sudo?

File: .circleci/config.yml (hunk 1 above)
Comment (knowbody) [reply .circleci/config.yml]: do I add this:
```
- restore_cache:
          keys:
            - dependencies-{{ checksum "package.json" }}
            - dependencies-
```

to fix it?

File: .circleci/config.yml (hunk 2 above)
Comment (knowbody) [reply .circleci/config.yml]: don't know, this is the first time I'm doing a CircleCI 2.0 config. I copied the one they provide for JS projects

Comment (knowbody) [reply .circleci/config.yml]: if you know how to fix, please push the changes

File: .circleci/config.yml (hunk 1 above)
Comment (czystyl) [reply .circleci/config.yml]: you can change `packages.json` to `yarn.lock` and done

**Pull request merged**